lib/modules/blockchain_process/blockchain.js

```diff
@@ -315,7 +315,7 @@ Blockchain.prototype.initChainAndGetAddress = function(callback) {
 };
 
 var BlockchainClient = function(blockchainConfig, client, env, onReadyCallback, onExitCallback) {
-  const isDev = blockchainConfig.isDev || blockchainConfig.default;
+  const isDev = Boolean(blockchainConfig.isDev);
```

Comment:
This can be coerced using `!!` as well. Not sure if using `Boolean()` is part of an "official" style guide or coding convention, so feel free to ignore my comment if that's the case :)

Reply:
ESLint doesn't like !! 😀